### PR TITLE
Nepal locales added

### PR DESCRIPTION
**Pull request by AshishMhrzn10**
I addded the locales in Nepali language for all continents and countries

### CODE REVIEW

**Comment by spielers** [comment]
Hey, @AshishMhrzn10 Thanks for your PR, I would suggest you make PR on the 0.7.x branch rather than the master branch.
You can find more detail in the issue description. #25

File: app/config/locales/np.countries.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    'UY' => 'उरुग्वे',
+    'US' => 'युनाइटेड अमेरिका',
+    'UZ' => 'उज्बेकिस्तान',
+    'VA' => 'भ्याटिकन सिटी',
+    'VC' => 'सेन्ट भिन्सेन्ट र ग्रेनाडाइन्स',
+    'VE' => 'भेनेजुएला',
+    'VN' => 'भियतनाम',
+    'VU' => 'भानुआटु',
+    'WS' => 'सामोआ',
+    'YE' => 'Yemen',
```
Comment (spielers) [inline app/config/locales/np.countries.php:194]: You can use the Nepali word  यमन instead of Yeman

File: app/config/locales/np.continents.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'AF' => 'अफ्रीका ',
+    'AN' => 'एन्टार्टिका ',
+    'AS' => 'एशिया',
+    'EU' => 'युरोप',
+    'NA' => 'उत्तर अमेरिका ',
+    'OC' => 'ओसिआनिया',
+    'SA' => 'दक्षिण  अमेरिका ',
```
Comment (spielers) [inline app/config/locales/np.continents.php:10]: remove the white spaces between

**Review by spielers — CHANGES_REQUESTED** [review]

**Comment by AshishMhrzn10** [comment]
i have done it but could u please help with pushing the git. I keep on
getting the error.Do we need to create a branch 0.7.x locally??

**Comment by spielers** [comment]
Not exactly you get help of GitHub desktop if your getting error with git
bash. When u clone the master branch all other repos are also get cloned to
your local system. Just head over to local system and start making changes
there.
Once done ✅ you can make PR from there

Best,
Spielers

File: app/config/locales/np.continents.php (hunk 2 above)
Comment (AshishMhrzn10) [reply app/config/locales/np.continents.php]: Please check issue description #623. I posted there succesfully.

**Comment by AshishMhrzn10** [comment]
Please check issue description #623. I posted there succesfully.

**Comment by spielers** [comment]
Now you can delete this PR @AshishMhrzn10 

Best,
spielers

**Pull request closed without merging**